src/getHumanAge.test.js

```diff
@@ -7,4 +7,58 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] if catAge = 0 and dogAge = 0', () => {
+    const actual = getHumanAge(0, 0);
+
+    expect(actual).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] if catAge < 15 and dogAge < 15', () => {
+    const actual = getHumanAge(11, 14);
+
+    expect(actual).toEqual([0, 0]);
+  });
+
+  test('should work only with positive numbers', () => {
+    const actual = getHumanAge(-2, -12);
+
+    expect(actual).toEqual([0, 0]);
+  });
+
+  test('should return NaN if input is not a number', () => {
+    const actual = getHumanAge('qwe', 15);
+
+    expect(actual).toEqual([NaN, 1]);
+  });
+
+  test('should return NaN if input is not passed', () => {
+    const actual = getHumanAge(15);
+
+    expect(actual).toEqual([1, NaN]);
+  });
```

Comment:
pay attention to the task description - ages are only positive numbers and integers. do not need to check string or negative numbers, or absence of the input parameters